core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -535,8 +537,6 @@ protected Scope visitRefreshMaterializedView(RefreshMaterializedView refreshMate
                     .map(ColumnMetadata::getName)
                     .collect(toImmutableList());
 
-            accessControl.checkCanInsertIntoTable(session.toSecurityContext(), targetTable);
```

Comment:
Why this changed? Separate commit?

Reply:
This is changed to ensure that we apply the refresh access control check on the MV object instead of insert access control on the storage table. I can move it to another commit though.